webapp/src/components/modals/create_issue/__snapshots__/create_issue_form.test.tsx.snap

```diff
@@ -1377,7 +1377,7 @@ exports[`components/CreateIssue should match snapshot with error 1`] = `
                     "self": "https://mmtest.atlassian.net/rest/api/2/issuetype/10002",
                   },
                   Object {
-                    "description": "The sub-task of the issue",
+                    "description": "Stories track functionality or features expressed as user goals.",
```

Comment:
Do you know why this snapshot changed so much?

Reply:
@mickmister some changes are related to the subtask and apart from that the order of file content is shuffled, mostly all the existing test cases are present in the new changes.